components/director/internal/domain/application/resolver.go

```diff
@@ -118,6 +123,7 @@ type Resolver struct {
 	webhookSvc  WebhookService
 	documentSvc DocumentService
 	sysAuthSvc  SystemAuthService
+	oauth20Svc  Oauth20Service
```

Comment:
```suggestion
	oAuth20Svc  OAuth20Service
```

Make sure to use proper casing in word `OAuth` across this PR.